open_samus_returns_rando/files/levels/s000_surface.lua

```diff
@@ -457,6 +457,9 @@ function s000_surface.OnSubAreaChange(_ARG_0_, _ARG_1_, _ARG_2_, _ARG_3_, _ARG_4
   elseif (_ARG_0_ == "collision_camera_024" and not Scenario.ReadFromBlackboard("alpha_killed", false)) then
     s000_surface.LaunchFirstTimeAlphaPresentation()
   end
+  if _ARG_0_ == "collision_camera_000" and _ARG_2_ == "collision_camera_017" then
+    Game.LoadScenario("c10_samus", "s110_surfaceb", "", "Samus", 1)
```

Comment:
Looks kinda scary to me to not define a start point. I'm even impressed that the game don't just move you to the void or to (0,0,0) coordinates.
If we ever glitch out by this teleport, we probably know why

Reply:
So what's happens is it warps to the scenario, and then since it doesn't have an actor listed, it just loads samus at coordinates where the load happened.  Since both Surfaces line up there, it's a perfect match. Going back into East I set coordinates because otherwise Samus warp position is inside the baby blocks, so this moves her outside the blocks barely.